pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerStatsRecorderImpl.java

```diff
@@ -143,11 +148,12 @@ private void init(ProducerConfigurationData conf) {
                     }
 
                     log.info("[{}] [{}] Pending messages: {} --- Publish throughput: {} msg/s --- {} Mbit/s --- "
-                            + "Latency: med: {} ms - 95pct: {} ms - 99pct: {} ms - 99.9pct: {} ms - max: {} ms --- "
-                            + "Ack received rate: {} ack/s --- Failed messages: {}", producer.getTopic(),
-                            producer.getProducerName(), producer.getPendingQueueSize(),
+                                    + "Latency: med: {} ms - 95pct: {} ms - 99pct: {} ms - 99.9pct: {} ms - max: {} ms --- "
+                                    + "Ack received rate: {} ack/s --- Failed messages: {}",
+                            producer.getTopic(),producer.getProducerName(),
+                            producer.getPendingQueueSize(),
                             THROUGHPUT_FORMAT.format(sendMsgsRate),
-                            THROUGHPUT_FORMAT.format(sendBytesRate / 1024 / 1024 * 8),
+                            THROUGHPUT_FORMAT.format(sendBytesRate * 8 / 1024 / 1024 ),
```

Comment:
maybe keep original `sendBytesRate / 1024 / 1024 * 8` is better here. it could avoid overflow

Reply:
got it, will fix it